### ISSUE TITLE
Keeping objects within `latent_individual` class

### ISSUE THREAD

**Issue by athowes** [epinowcast/epidist#272]
In #52 I've found that after making some alterations to an object of class `latent_individual` that I need to recall `as_latent_individual` in order for `epidist::epidist` to work:

```r
obs_prep_reduced <- obs_prep |>
  dplyr::group_by(district) |>
  dplyr::mutate(n = dplyr::n()) |>
  dplyr::filter(n > 30) |>
  dplyr::select(-n) |>
  dplyr::ungroup()

obs_prep_reduced <- as_latent_individual(obs_prep_reduced)
```

This doesn't seem to be ideal. Probably something easy to fix here.

**Comment by seabbs** [comment]
Have a look at the class system in `scoringutils` as I think this has been resolved their by @Bisaloo

**Comment by athowes** [comment]
Maybe can be fixed by #329.

**Comment by athowes** [comment]
Closing as duplicate of #399.
